Automattic/wp-calypso
# Playwright: implement ability to log in to WPCOM from a popup.

#### Changes proposed in this Pull Request

This PR implements the ability to log in to WPCOM through a popup window and the last bit of the original `wp-likes` spec.

In addition, some longstanding todo items have also been addressed in this patch:
- migrate `LoginPage` to be an inheritance of `BaseContainer`.
- implement a separate `getAccountCredential` and `getAccountSiteURL`.

#### Testing instructions

As usual:

CI

- ensure tests pass without any issues on TeamCity.

Local

- pull the branch and ensure they run with mocha --config .mocharc_playwright.yml specs/specs-playwright/.

Parent: #53169
